Formula/cdktf.rb

```diff
@@ -22,6 +22,9 @@ class Cdktf < Formula
   def install
     system "npm", "install", *Language::Node.std_npm_install_args(libexec)
     bin.install_symlink Dir["#{libexec}/bin/*"]
+
+    output = Utils.safe_popen_read("#{bin}/cdktf", "completion", "--help")
```

Comment:
Doesn't this just output the help text?

Reply:
from testing, no, it seems like `--help` outputs the completion only

Reply:
it accepts no arguments, however there's currently a bug causing strange behavior that this works around: https://github.com/hashicorp/terraform-cdk/issues/1905

it also supports both bash + zsh completions [by reading `SHELL`](https://github.com/yargs/yargs/blob/db354232705623bbcd8fad362f6a4d6d59650be5/lib/completion.ts#L44-L46), so we should add both. can be modeled after doitlive:

https://github.com/Homebrew/homebrew-core/blob/d034ddd0652e47f89a6e80a73d87c30d7f7913db/Formula/doitlive.rb#L57

Reply:
addressed in 36e0da6e0c4aec900d94d1acd98161a7379abb2d

Reply:
addressed in 4bf5a078330dd4abb156ec6b7c7209996761f340